src/schema/v2/artwork/artworkConsignmentSubmissionType.ts

```diff
@@ -23,6 +23,24 @@ const ArtworkConsignmentSubmissionType = new GraphQLObjectType<
           return statusDisplayTexts[consignmentSubmission.state]
         },
       },
+      inProgress: {
+        type: GraphQLBoolean,
+        resolve: (consignmentSubmission) => {
+          const inProgressSubmissionStates = [
+            "submitted",
+            "published",
+            "approved",
+            "hold",
+            "open",
+          ]
+
+          return inProgressSubmissionStates.includes(
+            consignmentSubmission.state
+          )
+            ? true
+            : false
```

Comment:
```suggestion
          return inProgressSubmissionStates.includes(
            consignmentSubmission.state
          )
```

Reply:
ahah, I'm idiot :) thanks for catching this